getting_started/step_by_step/ui_game_user_interface.rst

```diff
@@ -4,8 +4,8 @@ Design the GUI
 ==============
 
 Now that you've nailed the basics, we're going to see how to build a
-Game User Interface (GUI) with reusable UI components: a life bar, an
-energy bar, and bomb and rupee counters. By the end of this tutorial,
+game Graphical User Interface (GUI) with reusable UI components: a life bar, an
```

Comment:
Uhm, no, the abbreviation GUI means "Graphical User Interface": [TheFreeDictionary](https://acronyms.thefreedictionary.com/GUI), [Wiktionary](https://en.wiktionary.org/wiki/GUI#Noun)
"Game User Interface" is abbreviated to "Game UI" usually, or "game GUI" if one wants to emphasize that it is "Graphical"

Reply:
Well the change is consistent with what you're saying, it fixes the previous "Game User Interface" to "game GUI" as you mentioned.

Reply:
Oh, my bad, sorry.